### PR TITLE
Add URL_SAFE_ALPHABET and PADDED_URL_SAFE_ALPHABET

### PR DESCRIPTION
**Pull request by benjaminoakes**

### CODE REVIEW

File: lib/base64.rb [hunk 1]
```diff
@@ -22,6 +22,11 @@
 # binary data into purely printable characters.
 
 module Base64
+  # The alphabet described in RFC 4648 and implemented by
+  # +urlsafe_encode64+ and +urlsafe_decode64+.
+  URL_SAFE_ALPHABET = "-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz"
+  PADDED_URL_SAFE_ALPHABET = "-0123456789=ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz"
```
Comment (benjaminoakes) [inline lib/base64.rb:28]: These characters are listed in ASCII order

File: lib/base64.rb [hunk 2]
```diff
@@ -22,6 +22,13 @@
 # binary data into purely printable characters.
 
 module Base64
+  # The alphabet described in RFC 4648 and implemented by
+  # +urlsafe_encode64+ and +urlsafe_decode64+.
+  URL_SAFE_ALPHABET = "-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz"
+  # The padded alphabet described in RFC 4648 and implemented by
+  # +urlsafe_encode64+ and +urlsafe_decode64+.
+  PADDED_URL_SAFE_ALPHABET = "-0123456789=ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz"
```
Comment (benjaminoakes) [inline lib/base64.rb:30]: These characters are listed in ASCII order

**Comment by nobu** [comment]
For what purpose?
And base64 has the upstream https://github.com/ruby/base64 .

**Comment by benjaminoakes** [comment]
Thank you! I was unaware. I'll make a pull request on that project with more details.

**Pull request closed without merging**